.github/workflows/benchmarking-workflow.yml

```diff
@@ -0,0 +1,60 @@
+name: Validate SPDX License year and company mention in currently changed Files
```

Comment:
I think this file should be renamed from `benchmarking-workflow.yml` to something more general or descriptive for this repo.

Reply:
This was just for testing the old workflow -- currently in the benchmarking repo , will be removed before merging the PR